packages/@lightningjs/ui-components/src/docs/ThemingMode.mdx

```diff
@@ -41,7 +37,7 @@ You can set any component's default mode by specifying it in the `componentConfi
 ```js
 componentConfig: {
   Tile: {
-    mode: 'focused'
+    mode: 'focused';
```

Comment:
remove the semi-colon since these are in objects

Reply:
the linter is auto correcting this every time I remove it. I'll have to dig into the linter for mdx files to see why